components/Header/Header.tsx

```diff
@@ -57,10 +74,62 @@ const Header: FC<HeaderProps> = () => {
           </Link>
         </span>
 
-        <div className="flex-row items-center text-sm rounded lg:items-right lg:w-auto lg:md:text-sm font-quicksand">
-          {isMobile && <MobileMenu />}
-          {!isMobile && <DesktopMenu />}
+        <div
+          className="text-sm font-quicksand 
+        flex flex-row items-center
+        md:gap-x-[25px] xl:gap-x-[40px]"
+        >
+          {!isHidden && !isMobile && (
+            <div className="flex items-center font-bold font-quicksand">
+              {!isLaptop && (
+                <button
+                  type="button"
+                  className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+                  onClick={() => onChangeThemeConfig("light")}
+                >
+                  light
+                </button>
+              )}
+              <ToggleButton
+                onClick={onToggle}
+                value={themeMode === "dark"}
+                id="light_dark_switch"
+              />
+              {!isLaptop && (
+                <button
+                  type="button"
+                  className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+                  onClick={() => onChangeThemeConfig("dark")}
+                >
+                  dark
+                </button>
+              )}
+            </div>
+          )}
+          {!isMobile && <DesktopExplore />}
+          {!isMobile && <DesktopCollections />}
+          {!isMobile && (
+            <>
+              <DiscordIcon />
+              <a href="https://twitter.com/Cre8orsNFT" target="_blank" rel="noreferrer">
+                <div className="cursor-pointer ">
+                  <Image
+                    src={`${
+                      themeMode === "dark" && !isMintPage
+                        ? "/assets/Header/white_twitter.png"
+                        : "/assets/Header/new_twitter.png"
+                    }`}
+                    width={19}
+                    height={19}
+                    alt="twitter"
+                  />
+                </div>
+              </a>
+              <CustomConnectWallet />
+            </>
+          )}
```

Comment:
we can combine this into one `!isMobile`

Reply:
fixed 👍